package/src/countmon.h

```diff
@@ -6,17 +6,19 @@
 
 #ifndef PRMON_COUNTMON_H
 #define PRMON_COUNTMON_H 1
+#define MY_NAME "countmon"
```

Comment:
I think we can define this in the source file instead (applies to other cases too) and un-define it after using. Also, we can perhaps call it something more explanatory, e.g. `MONITOR_NAME`.

Reply:
If it's going to be private maybe should it be something like `THIS_MONITOR_NAME_` (or were you thinking of calling it, e.g., `COUNTMON_NAME_` (different in each monitor).

I agree that undefining it at the end is smart - stops it from leaking out.